gdcdictionary/schemas/treatment.yaml

```diff
@@ -3959,7 +3972,6 @@ properties:
       - PARP Inhibitor NMS-03305293
       - PARP-1/2 Inhibitor ABT-767
       - PARP/Tankyrase Inhibitor 2X-121
-      - PARP/Tankyrase Inhibitor 2X-121
```

Comment:
Now that these duplicates are being removed, one line needs to be removed from a unit test. Delete this line:
https://github.com/NCI-GDC/gdcdictionary/blob/7107e8116ce6ed8185626570dcba14b46e8e4d27/tests/schema_test.py#L62

That should make the tests pass for this PR